src/stream_reader.h

```diff
@@ -23,10 +23,42 @@
 
 class StreamReader {
 public:
-    StreamReader() = default;
+    StreamReader(vsag::Allocator* allocator) : buffer_(allocator) {
```

Comment:
buffer_(allocator) looks like `buffer_ = allocator` :)

Reply:
This is the invocation of the constructor of buffer_ to initialize the alignment.